volume-group-operator/controllers/volumegroup_controller.go

```diff
@@ -18,19 +18,31 @@ package controllers
 
 import (
 	"context"
+	"time"
 	"github.com/IBM/volume-group-operator/pkg/config"
 	"github.com/go-logr/logr"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/log"
+	log "sigs.k8s.io/controller-runtime/pkg/log"
 
 	volumegroupv1 "github.com/IBM/volume-group-operator/api/v1"
 	grpcClient "github.com/IBM/volume-group-operator/pkg/client"
 )
 
+var volumeGroupLog = log.Log.WithName("volumegroup_controller")
+
+const (
+	VolumeGroup               = "VolumeGroup"
+	VolumeGroupClass          = "VolumeGroupClass"
+	VolumeGroupContent        = "VolumeGroupContent"
+)
```

Comment:
can you move it settings file in this package?

Reply:
Following our conversation, I will move these consts into settings,go file that is waiting to be merged in PR https://github.com/IBM/ibm-block-csi-operator/pull/294